examples/support/mock_data.py

```diff
@@ -147,15 +147,20 @@ def _generate_galaxy_catalog(cluster_m, cluster_z, cluster_c, cosmo, ngals, Delt
                                             delta_mdef=Delta_SO, halo_profile_model='nfw',
                                             z_src_model='single_plane')
     galaxy_catalog['gammat'] = gamt
+    galaxy_catalog['gammax'] = np.zeros(ngals)
 
     # Add shape noise to source galaxy shears
     if shapenoise is not None:
         galaxy_catalog['gammat'] += shapenoise*np.random.standard_normal(ngals)
+        galaxy_catalog['gammax'] += shapenoise*np.random.standard_normal(ngals)
 
     # Compute ellipticities
     galaxy_catalog['posangle'] = np.arctan2(galaxy_catalog['y_mpc'], galaxy_catalog['x_mpc'])
-    galaxy_catalog['e1'] = -galaxy_catalog['gammat']*np.cos(2*galaxy_catalog['posangle'])
-    galaxy_catalog['e2'] = -galaxy_catalog['gammat']*np.sin(2*galaxy_catalog['posangle'])
+
+    galaxy_catalog['e1'] = -galaxy_catalog['gammat']*np.cos(2*galaxy_catalog['posangle']) \
+                           + galaxy_catalog['gammax']*np.sin(2*galaxy_catalog['posangle'])
+    galaxy_catalog['e2'] = -galaxy_catalog['gammat']*np.sin(2*galaxy_catalog['posangle']) \
+                           + galaxy_catalog['gammax']*np.cos(2*galaxy_catalog['posangle'])
```

Comment:
Hi @combet , sorry, could you check this line? I feel like there is a minus sign here (line 163)...

Reply:
@shenmingfu, you're right, thanks for checking this out. It's corrected now.